Astro.podspec

```
@@ -30,8 +30,8 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
   }
   s.source           = { :git => "https://github.com/RobotsAndPencils/Astro.git", :tag => s.version.to_s }
 
-  s.ios.deployment_target = '9.0'
-  s.platform     = :ios, '9.0'
+  s.ios.deployment_target = '11.0'
+  s.platform     = :ios, '12.0'
```

Are these two lines supposed to be different OS versions? When I checked [the docs](https://guides.cocoapods.org/syntax/podspec.html#platform) it looks like since we only target iOS we can remove `deployment_target` line, and based on your other changes in this PR the OS version should probably be `11.4`?

```suggestion
  s.platform     = :ios, '11.4'
```

---

Ok. I just assumed platform was the version you compiled against and deployment target was the one targeted for deployment. Just like in Xcode.

---

done

---

Yeah this will just be the deployment target (which corresponds with the value in a users' Podfile) and the SDK will be determined by the Xcode that users integrating Astro use to build their app.